app/modules/FrontModule/Registration/components/RegistrationForm/RegistrationForm.php

```diff
@@ -105,12 +105,12 @@ public function createComponentRegistrationForm(): Form
         $form->addSubmit('actionSend', 'Save');
 
         $form->onValidate[] = function () use ($form): void {
-            $values = $form->getValues();
-
-            if ($values->email !== '') {
+            if ($form['email']->getValue() !== '') {
```

Comment:
tohle prochazi pres PHPStan ? (nevim nezkousel jsem, ale mel jsem za to ze jsme mazali array access kvuli PHPStan

Reply:
Máš pravdu, neprojde to. Opravím a udělám nový Pull Request. Nicméně TRAVIS padá i na tom, že si PHPStan myslí, že metoda $form->getValues() vrací ArrayHash, tedy volání metody 'processForm()' se třídou 'Registration' se mu nelíbí. A toto nevím jak opravit.

Reply:
Nemusíš dělat nový PR, můžeš udělat změnu a následně force-push do forku a automaticky se to promítne do tohoto PR :) A Travis zařídí nový build

Reply:
> Nemusíš dělat nový PR, můžeš udělat změnu a následně force-push do forku a automaticky se to promítne do tohoto PR :) A Travis zařídí nový build

Boží to jsem nevěděl. Díky. Něco se u toho i naučím. :)